docker/bin/stop-presto

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# This works for worker only. Coordinator doesn't support graceful shutdown.
+# This script will block until the server has actually shutdown.
+set -x
+
+http_port="$(cat /usr/lib/presto/etc/config.properties | grep 'http-server.http.port' | sed 's/^.*=\(.*\)$/\1/')"
+https_port="$(cat /usr/lib/presto/etc/config.properties | grep 'http-server.https.port' | sed 's/^.*=\(.*\)$/\1/')"
+
+if [ -n "$http_port" ] ; then
+    res=$(curl -s -o /dev/null -w "%{http_code}"  -XPUT --data '"SHUTTING_DOWN"' -H "Content-type: application/json" http://localhost:${http_port}/v1/info/state)
+fi
+
+if [ -z "$res" -o "$res" != "200" ] && [ -n "$https_port" ]; then
```

Comment:
- `res` can be undefined at this point
- also rename `res` to something more meaningful
- if you use curl `--fail`, no need to check for 200

Reply:
Fixed